common/src/main/java/com/movie/recommender/common/model/movie/NowPlayingMovieDetail.java

```diff
@@ -0,0 +1,20 @@
+package com.movie.recommender.common.model.movie;
+
+import lombok.Data;
+
+
+import java.util.List;
+
+@Data
+public class NowPlayingMovieDetail {
```

Comment:
there is no need to create a new type for this
you already have the `MovieDetails` class, which has the same structure
for the now-playing list, we can ignore genres, cast, crew, and keywords
delete this class and use `MovieDetails` in `NowPlayingMoviesByCountry`